File: app/src/main/java/com/example/bmi_calc/GraphActivity.java
package com.example.bmi_calc;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;

public class GraphActivity extends AppCompatActivity {
    private Button buttonBack;

    @Override
    protected void onCreate(Bundle savedInstanceState) {

        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_graph);

        buttonBack = findViewById(R.id.buttonBack);

        Button buttonBack = findViewById(R.id.buttonBack);
        buttonBack.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                finish();
            }
        });

    }

}

File: app/src/main/java/com/example/bmi_calc/GraphActivity.kt
package com.example.bmi_calc

import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class GraphActivity : AppCompatActivity() {
    private var buttonBack: Button? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_graph)
        buttonBack = findViewById(R.id.buttonBack)
        val buttonBack = findViewById<Button>(R.id.buttonBack)
        buttonBack.setOnClickListener { finish() }
    }
}